### PR TITLE
*: pin go-jsonnet binary to v0.12.1 version

### PR DESCRIPTION
**Pull request by paulfantom**
Do not use go-jsonnet from master branch as this might contain problematic code (as seen in https://github.com/openshift/cluster-monitoring-operator/pull/349). Instead of installing binary with `go get` use `git checkout` and install version v0.12.1.

### CODE REVIEW

**Comment by metalmatze** [comment]
It would be really nice to have pre-build jsonnet binaries at some point. :man_shrugging:

**Comment by paulfantom** [comment]
Follow-up with pre-built binaries shipped in a container: https://github.com/openshift/cluster-monitoring-operator/pull/351

File: assets/alertmanager/secret.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  alertmanager.yaml: Imdsb2JhbCI6CiAgInJlc29sdmVfdGltZW91dCI6ICI1bSIKInJlY2VpdmVycyI6Ci0gIm5hbWUiOiAibnVsbCIKInJvdXRlIjoKICAiZ3JvdXBfYnkiOgogIC0gImpvYiIKICAiZ3JvdXBfaW50ZXJ2YWwiOiAiNW0iCiAgImdyb3VwX3dhaXQiOiAiMzBzIgogICJyZWNlaXZlciI6ICJudWxsIgogICJyZXBlYXRfaW50ZXJ2YWwiOiAiMTJoIgogICJyb3V0ZXMiOgogIC0gIm1hdGNoIjoKICAgICAgImFsZXJ0bmFtZSI6ICJXYXRjaGRvZyIKICAgICJyZWNlaXZlciI6ICJudWxsIg==
+  alertmanager.yaml: Imdsb2JhbCI6IAogICJyZXNvbHZlX3RpbWVvdXQiOiAiNW0iCiJyZWNlaXZlcnMiOiAKLSAibmFtZSI6ICJudWxsIgoicm91dGUiOiAKICAiZ3JvdXBfYnkiOiAKICAtICJqb2IiCiAgImdyb3VwX2ludGVydmFsIjogIjVtIgogICJncm91cF93YWl0IjogIjMwcyIKICAicmVjZWl2ZXIiOiAibnVsbCIKICAicmVwZWF0X2ludGVydmFsIjogIjEyaCIKICAicm91dGVzIjogCiAgLSAibWF0Y2giOiAKICAgICAgImFsZXJ0bmFtZSI6ICJXYXRjaGRvZyIKICAgICJyZWNlaXZlciI6ICJudWxsIg==
```
Comment (s-urbaniak) [inline assets/alertmanager/secret.yaml]: this is funky, but I verified that structurally there is no difference. Apparently the newer version of go-jsonnet adds a space character after decoding the above with base64:

old `"global":<LF>`:
```
00000000: 2267 6c6f 6261 6c22 3a0a 2020 2272 6573  "global":.  "res
...
```
new `"global":<SP><LF>`
```
00000000: 2267 6c6f 6261 6c22 3a20 0a20 2022 7265  "global": .  "re
```

Comment (squat) [reply assets/alertmanager/secret.yaml]: this was driving me nuts for all of my PRs

**Pull request merged**